### ISSUE TITLE
Some examples don't work

### ISSUE THREAD

**Issue by Rudxain** [bjpop/js-turtle#8]
# Tested Browsers
MS Edge & G Chrome (both non-beta/nightly and up-to-date)

# Steps to reproduce
0. [Go to the website](http://htmlpreview.github.io/?https://github.com/bjpop/js-turtle/blob/master/turtle.html), or [download the repo](https://github.com/bjpop/js-turtle/archive/refs/heads/master.zip) and open the root `index.html`
1. Select all the text within the `examples/bounce.js` file, and copy
2. Paste it on the "Definitions" box in the website (while removing the prepopulated text)
3. Run `demo()` command and hit "Enter"
4. The canvas goes blank, and the turtle is nowhere to be found

# About the other files
* `clock` is fine
* `randstripe` has the same problem as `bounce`
* `sierpinski` works as expected
* `spiral`: same problem

# Etc
I'm currently trying to pinpoint the cause of the bug

**Comment by Rudxain** [comment]
The `rain` fn in `bounce` has 4 conditions that map to 2 unique cases. In other words, they execute the same statements even when they seem like they shouldn't. I suppose the difference should be the sign inversion, since that's the thing that all 4 cases have in common. I tried adding and removing the `-` for each of the 4 cases, but to no avail, it still doesn't work.

Edit: I realized it seems that's intentional? I'm now refactoring to merge conditions to their matching blocks
